tests/Avalonia.Markup.Xaml.UnitTests/Xaml/StyleTests.cs

```diff
@@ -739,5 +737,28 @@ public void Fails_Use_Classes_In_Setter_When_Selector_Is_Complex()
                 Assert.Equal ("Cannot set Classes Binding property '(Classes.Banned)' because the style has an activator. Line 6, position 14.", exception.Message);
             }
         }
+
+        [Theory]
+        [InlineData("<Style>", "</Style>")]
+        [InlineData("<Style Selector=''>", "</Style>")]
+        [InlineData("<Styles><Style>", "</Style></Styles>")]
+        [InlineData("<Styles><Style Selector=''>", "</Style></Styles>")]
+        public void No_Selector_Should_Target_Parent_Type(string styleStart, string styleEnd)
+        {
+            using var app = UnitTestApplication.Start(TestServices.StyledWindow);
+
+            var window = (Window)AvaloniaRuntimeXamlLoader.Load(
+                $"""
+                <Window xmlns="https://github.com/avaloniaui">
+                    <Window.Styles>
+                        {styleStart}
+                            <Setter Property="WindowStartupLocation" Value="CenterScreen" />
+                        {styleEnd}
+                    </Window.Styles>
+                </Window>
+                """);
+
+            Assert.Equal(WindowStartupLocation.CenterScreen, window.WindowStartupLocation);
```

Comment:
Use `Tag` or something less dependent on default values being changed?

Reply:
Good point! I wanted a property from `Window` itself, to prove that we're not falling back to `Control` or equivalent. Changed to `Window.Title`.